python/lsst/ctrl/oods/gen2ButlerBroker.py

```diff
@@ -0,0 +1,90 @@
+# This file is part of ctrl_oods
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import logging
+import os
+import shutil
+from lsst.ctrl.oods.butlerBroker import ButlerBroker
+from lsst.pipe.tasks.ingest import IngestTask
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Gen2ButlerBroker(ButlerBroker):
+    """Processes files on behalf of a Gen2 Butler.
+
+    Parameters
+    ----------
+    butlerConfig: `dict`
+        dictionary containing butler configuration information
+    """
+    def __init__(self, butlerConfig, publisher, publisher_queue):
```

Comment:
Parameters `publisher` and `publisher_queue` are not described in the docstring.

Reply:
Added